src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -55,6 +56,8 @@ private static class LoadingContext {
         private final Set<ShuntCompensator> shuntSet = new LinkedHashSet<>();
 
         private final Set<HvdcLine> hvdcLineSet = new LinkedHashSet<>();
+
+        private final Map<Terminal, LfArea> areaBoundaries = new HashMap<>();
```

Comment:
Naming is still confusing beacause not all those terminals are boundaries of the area (because of dangling lines). 
Name it areaTerminalMap instead?

Reply:
Yes I agree, I renamed it in 5a29ee99ce070c764b03988fb43c2636eb909798